scaleway/helpers.go

```diff
@@ -848,29 +848,87 @@ func retryOnTransientStateError[T any, U any](action func() (T, error), waiter f
 	return t, err
 }
 
+// compareLocalities compare two localities
+// They are equal if they are the same or if one is a zone contained in the other
```

Comment:
I'm not sure to understand this sentence. What does it mean for a zone to be contained in another zone? Are you talking about region and zone comparison?

Reply:
Comment have been improved